src/app/frontend/_rtl.scss

```diff
@@ -17,18 +17,21 @@ body:lang(fa) {
   direction: rtl;
 
   .mat-form-field {
-    text-align: unset !important;
+    text-align: unset;
   }
   .mat-paginator-range-actions {
     flex-direction: row-reverse;
   }
   .kd-slider-value {
-    text-align: left !important;
+    text-align: left;
   }
   .mat-option {
     text-align: right;
-    padding: 0 16px !important;
-    padding-right: 32px !important;
+  }
+  .mat-optgroup .mat-option {
+
```

Comment:
Focus on code style and spaces. See how it is done in other places.

Reply:
I think I fixed them. Please see my lastest [commit](https://github.com/kubernetes/dashboard/pull/5701/commits/88a43e02c1f2b9464cc53365e49b4ed33c506a92).